File: remove_rom.pl
#!/usr/bin/perl -w

$file = 'pacemu-master.dsk';
open FILE, "<$file" and binmode FILE or die "$file: $!\n";
read FILE, $data='', -s $file;
close FILE;

$file = 'disk.base';
open FILE, ">$file" and binmode FILE or die "$file: $!\n";
print FILE substr $data, 0, 819200-16384;
close FILE;

File: remove_rom.py
#!/usr/bin/env python

import argparse

parser = argparse.ArgumentParser("Remove ROM placeholder from pacemu master disk image")
parser.add_argument("diskimage")
parser.add_argument("baseimage")
args = parser.parse_args()

with open(args.diskimage, 'rb') as infile:
    diskdata = infile.read()

with open(args.baseimage, 'wb') as outfile:
    outfile.write(diskdata[0:819200-16384])
